lib/virility/strategy.rb

```diff
@@ -52,6 +53,7 @@ def call_strategy
     #
 
     def collect_results
+      binding.pry
```

Comment:
Maybe remove  this ?? !

Reply:
This was an intermediate commit, the line was removed